src/main/java/tigase/server/xmppclient/StreamManagementIOProcessor.java

```diff
@@ -748,13 +763,32 @@ public void ack(int value) {
 				count = (Integer.MAX_VALUE - value) + get() + 1;
 			}
 
+			lastConfirmationAt = System.currentTimeMillis();
+
 			while (count < queue.size()) {
 				queue.poll();
 			}
 		}
 
+		/**
+		 * Method notifies class that request for ack is being sent
+		 */
+		public void sendingRequest() {
+			lastRequestSentAt = System.currentTimeMillis();
+			lastRequestSentFor = get();
+		}
+
+		/**
+		 * Sets ack request count value
+		 * @param ackRequestCount
+		 */
+		public void setAckRequestCount(int ackRequestCount) {
+			this.ackRequestCount = ackRequestCount;
+		}
+
+		@Deprecated
+		@TigaseDeprecated(removeIn = "9.0.0", since = "8.3.0", note = "Method will not be called any more")
```

Comment:
why?

Reply:
As it was setting a variable which was never used. Keeping this is pointess?